### PR TITLE
AddFakeAandRefactorEditEmailTest

### PR DESCRIPTION
**Pull request by AdamWilkanowski**

### CODE REVIEW

File: cypress/integration/customizedResources/user/add/AddRecord.spec.js [hunk 1]
```diff
@@ -11,22 +11,21 @@ import { getFormValues,
 } from '../../../../support/helpersMethods';
   
 const { inputs, buttons, boardView } = customized;
-const { inputsTexts  } = common;
-  
+let email = 'aaaaaa' + faker.internet.email().toLowerCase();
```
Comment (areksb) [inline cypress/integration/customizedResources/user/add/AddRecord.spec.js]: is that `toLowerCase()` necessary?

File: cypress/integration/mongooseResources/category/delete/DeleteRecord.spec.js [hunk 2]
```diff
@@ -23,6 +23,8 @@ describe('Delete category record',function(){
       .get(boardView.table).then($tableWithRecords=>{ 
         const firstRecordIdAfeterDelete = $tableWithRecords.find(boardView.tableTds).eq(2).text();
         expect(this.firstRecordId.text()).not.to.equal(firstRecordIdAfeterDelete);
+       
+
```
Comment (areksb) [inline cypress/integration/mongooseResources/category/delete/DeleteRecord.spec.js]: probably unnecessary?

File: cypress/integration/customizedResources/user/add/AddRecord.spec.js (hunk 1 above)
Comment (AdamWilkanowski) [reply cypress/integration/customizedResources/user/add/AddRecord.spec.js]: Necessary, Admin automatically makes email to lower cases, so I have to do this in the test also. Without it, the test fails.
e.g.
'aaaaaasigurd2@gmail.com' to deeply equal 'aaaaaaSigurd2@gmail.com'  -Assertion failed

File: cypress/integration/mongooseResources/category/delete/DeleteRecord.spec.js (hunk 2 above)
Comment (AdamWilkanowski) [reply cypress/integration/mongooseResources/category/delete/DeleteRecord.spec.js]: Unnecessary

**Pull request merged**